infrastructure/nomad-configuration/client-instance-user-data.tpl.sh

```diff
@@ -46,10 +46,11 @@ while [  $COUNTER -lt 99 ]; do
 done
 
 sleep 15
-ATTACHED_AS=`lsblk -n | grep 500G | cut -d' ' -f1`
+ATTACHED_AS=`lsblk -n | grep 8.8T | cut -d' ' -f1`
```

Comment:
If you're going to change this value in the code can you also change it in the TF